src/test/resources/features/complaints/ComplaintsTriage.feature

```diff
@@ -309,7 +309,7 @@ Feature: Complaints Triage
     And I submit details on the Triage Capture Reason page
     And I send the case to drafting
     Then the case should be moved to the "DRAFT" stage
-    And the summary should display the owning team as "Border Force"
+    And the summary should display the owning team as "Border Force (Stage 2)"
```

Comment:
Think this is a mistake

Reply:
Yeah, I accidentally changed the wrong scenarios when I realised stage 2 had it's own team